### ISSUE TITLE
`/subscriptions/sites`: Unfollow site seems slow, try setting the item on 50% opacity after clicking unfollow button until the request is finished.

### ISSUE THREAD

**Issue by yansern** [Automattic/wp-calypso#75220]
When clicking on the Unfollow button, there seems to be a delay before the item is removed from the screen (at least the delay is happening when the request takes longer to complete).

I'd like to propose:
- after clicking the unfollow button, add a class that sets the item to around 50% opacity.
- once the request is finished, the item should be removed by itself already.
- if the request failed, remove the class that sets the item to around 50% opacity.
- nice to have: show error message when unfollow fail. https://github.com/Automattic/wp-calypso/issues/75222
